eslintrc.json

```diff
@@ -1,21 +1,9 @@
 {
-  "extends": "standard",
+  "extends": "airbnb",
   "rules": {
-    "arrow-body-style": 2,
     "arrow-parens": [2, "as-needed"],
```

Comment:
[airbnb disables this rule](https://github.com/airbnb/javascript/blob/4ef335e24276a6c2a46fe7a0674f49cfcff79796/packages/eslint-config-airbnb/rules/es6.js#L29). Leaving it enabled for us so that parens are only required as needed